packages/studio-plugin/src/createStudioStylingPlugin.ts

```diff
@@ -40,7 +40,7 @@ function isImportedByStudio(importer: string) {
   const unixImporter = upath.toUnix(importer);
   return !!(
     unixImporter.match(/.*\/node_modules\/@yext\/studio.*/) ||
-    importer.startsWith(packagesDir)
+    unixImporter.startsWith(packagesDir)
```

Comment:
I was concerned that on windows, the `importer` would be in windows, and the `packageDir` would be in unix.  So just to be safe, I converted them both to unix.  Do you think this is unnecessary?

I tried to find info about what `upath.resolve` does exactly but for some reason I was having trouble tracing it back...